### PR TITLE
Fixes to README.md

### PR DESCRIPTION
**Pull request by JEHoctor**
A number of small problems seems to have entered README.md due to things like changing the directory structure of the project and putting the project on Github. (I think Github parses markdown differently than Gitlab.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,17 +135,17 @@ And for `print_number`, we'll use `processopedia -t print_number -d`:
...
+One KWIVER's great strengths (as provided by Sprokit) is the ability to create hybrid pipelines which combine C++ and Python processes in the same pipeline.  This greatly facilitates prototyping complex processing pipelines.  To test this out we'll still use the `numbers` process, but we'll use a Python version of the `print_number` process called `kw_print_number_process` the code for which can be seen in [sprokit/processes/python/kw_print_number_process.py](sprokit/processes/python/kw_print_number_process.py).    As usual, we can lean about this process with a `processopedia` command: `processopedia -t kw_print_number_process -d`:
```
Comment (Purg) [inline README.md]: Not your fault, but might as well fix it while we're in here: "One [of] KWIVER's..."

File: README.md [hunk 2]
```diff
@@ -238,9 +238,7 @@ turned ON or OFF through CMake variables.
 
 All dependencies are supplied by the Fletch package of 3rd party dependencies.
 
-[Eigen](http://eigen.tuxfamily.org/) (>= 3.0)
-[log4cxx] (https://logging.apache.org/log4cxx/) (>= 0.10.0)
-[Apache Runtime] (https://apr.apache.org/)
+[Eigen](http://eigen.tuxfamily.org/) (>= 3.0) [log4cxx](https://logging.apache.org/log4cxx/) (>= 0.10.0) [Apache Runtime](https://apr.apache.org/)
```
Comment (Purg) [inline README.md]: Was it intentional that these all got jammed into one line? I would argue for them to be separate lines if the one-lining was on purpose.

Comment (JEHoctor) [inline README.md]: No, that wasn't intentional I just wanted to fix the way the links displayed with Github's markdown parsing. Multiple lines makes sense to me.

File: README.md (hunk 1 above)
Comment (JEHoctor) [inline README.md]: Yup, I'll add this fix.

**Comment by linus-sherrill** [comment]
Could you merge these changes to kitware:master rather than release?

**Comment by mleotta** [comment]
Actually, now that I've updated the release branch, we probably should merge these into  kitware:release

**Pull request merged**